### DISCUSSION TITLE
Secrets for v4.3.0

### DISCUSSION THREAD

**Discussion by stuartoram** [mastodon/mastodon#33110] [Q&A]
Apologies for the simple question, but I haven't got around to attempting the 4.3.x upgrade yet, as I haven't worked out the steps to generate the secrets and where/how to set the variables. If someone could give some pointers in the right direction, I'm sure I'll then be able to work it out from there. Thanks!

**Comment by StrangeMinds1960** [comment]
The instructions can be found here (scroll down quite a long way)

https://github.com/mastodon/mastodon/releases/tag/v4.3.0

It's a while since I did it, but I think you just paste the output of bin/rails db:encryption:init into .env.production
